schematron/v2.2.0/v2-2-0_L200_Audit.csv

```diff
@@ -216,6 +216,7 @@
 "","","","","","","","","","auc:ReheatSource/text() = 'None' or auc:ReheatControlMethod","","",""
 "","","","","","","","","","auc:ReheatSource/text() != 'Heating plant' or auc:ReheatPlantID","","",""
 "","","Central Fan","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Systems/auc:HVACSystems/auc:HVACSystem/auc:HeatingAndCoolingSystems/auc:Deliveries/auc:Delivery/auc:DeliveryType/auc:CentralAirDistribution[auc:AirDeliveryType/text() = 'Central fan']","false","deliveryID","current()/ancestor::auc:Delivery/@ID","//auc:Systems/auc:FanSystems/auc:FanSystem[auc:LinkedSystemIDs/auc:LinkedSystemID/@IDref = $deliveryID]","auc:Delivery ID must be linked to a valid auc:FanSystem","",""
```

Comment:
Hey @macintoshpie, so I think we can remove the predicate `[auc:AirDeliveryType/text() = 'Central fan']`.  Per our conversation https://github.com/BuildingSync/TestSuite/issues/38, we are just looking at **any** delivery that is an `auc:CentralAirDistribution`

Reply:
Addressed here b0bb041193012a6cc2713702071b47d292afbd1b

Reply:
🌭